Automattic/wp-calypso
# Reader: remove old unused feed header component

To Test:
- visit feeds + blogs and make sure they still have the right header
- that there is exactly one back button on feeds
